packages/devextreme/js/__internal/viz/chart_components/rolling_stock.ts

```diff
@@ -0,0 +1,106 @@
+interface BoundingRect {
+  x: number;
+  y: number;
+  width: number;
+  height: number;
+}
+
+interface RollingStockLabel {
+  getBoundingRect: () => BoundingRect;
+  hideInsideLabel: (coords: { x: number; y: number }) => boolean;
+  getData: () => { value: number | string | Date };
```

Comment:
i mb missed something. Whats the point of object having 1 field? can't we return value right away?